lib/deployment.js

```diff
@@ -137,7 +142,7 @@ class Deployment extends EventEmitter {
    * @return {array}
    */
   activeTasks() {
-    return _.difference(this.tasksStarted, this.tasksFailed);
+    return Deployment._difference(this.tasksStarted, this.tasksFailed);
```

Comment:
so here if we are returning some tasks that failed as well, then it's not actually the active tasks that we are trying o fetch.

Reply:
You were completely correct. I've fixed this now.